sections/compare-blender.liquid

```diff
@@ -1,19 +1,240 @@
+<script>
+  var standardize_cell_size = function(){};
+</script>
+
+<div style="display:none;" onclick="exitPopup()" id="popupBackground"></div>
+<div id="thePopup" style="display:none;">
+<button onclick="exitPopup()" type="button" id="exitButton" class="close product-comparison--close" data-dismiss="modal" aria-hidden="true" style="
+    background: transparent;
+    padding: 0;
+    border: 0;
+    position:absolute;
+    cursor:pointer;
+    right:10px;
+    top:10px;
+">
+<i class="bt-icon-action svg product-comparison--icon-close">
+  <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 1024 1024" enable-background="new 0 0 1024 1024" xml:space="preserve" style="
+    width: 25px;
+    height: 25px;
+">
+  <circle id="circle" fill="none" stroke="none" stroke-width="10" stroke-miterlimit="10" cx="512" cy="511.1" r="503.9" style="
+    stroke-dasharray: 0;
+    stroke-width: 57;
+    stroke: #fff;
+">
+  </circle>
+  <path id="arrow" fill="none" d="M421.5,206.3l-72.7,72.7L587.7,518L348.8,756.9l72.7,72.7L733.1,518
+    L421.5,206.3z"></path>
+  <polygon id="plus" fill="none" points="827.7,472.4 552,472.4 552,196.7 473.2,196.7 473.2,472.4 197.5,472.4 197.5,551.1
+    473.2,551.1 473.2,826.8 552,826.8 552,551.1 827.7,551.1 "></polygon>
+  <rect id="minus" x="197.5" y="472.4" fill="none" width="630.2" height="78.8">
+  </rect>
+  <polygon id="close" fill="none" points="762.2,706.7 567.3,511.8 762.2,316.8 706.5,261.1 511.6,456.1 316.6,261.1 261,316.8
+  455.9,511.7 261,706.7 316.7,762.4 511.6,567.4 706.6,762.4 " style="
+    fill: #fff;
+"></polygon>
+  </svg>
+
+  <div class="svg-action-icon-alternate"></div>
+</i>        </button>
+
+<div>
+  <img id="zoomInProduct" style="width:100%;">
+</div>
+<div>
+  <img id="zoomInPanel" style="width:100%;">
+</div>
+<div>
+
+</div>
+</div>
+
+<div class="full-width layout wrap product-comparison--bg-image">
+      <div class="row-fluid top-level head-image">
+        <div class="col-md-12 product-comparison--header-container">
+          <h1 class="product-comparison--leader">{{ section.settings.product_compare_title }}</h1>
+          <h2 class="product-comparison--sub-leader">{{ section.settings.product_compare_title_2 }}</h2>
+        </div>
+      </div>
+    </div>
+
+<div style="max-width:80rem;margin:0 auto;">
+<h3 style="margin-left:20%;margin-top:50px;">{{ section.settings.product_row_1_title }}</h3>
+
+
+
+<div class="grid product-table" style="margin-left:0px;">
 
-<div class="grid">
   {% for i in section.blocks %}
-    {% assign product_name = i.settings.product_id | text  %}
-    {% include 'product-to-compare' %}
+    {% if i.type == 'products-1' %}
+      {% assign product_name = i.settings.product_id | text  %}
+      {% include 'product-to-compare' %}
+    {% endif %}
   {% endfor %}
 </div>
 
+
+
+<h3 style="margin-left:20%;">{{ section.settings.product_row_2_title }}</h3>
+
```

Comment:
Should this inline style be part of the product-table or should this receive a distinct class? we're kind of mixing and matching thing.

Reply:
See what you can do about all the inline styles. I'd like to respect the separation of concern for all the different presentation layer components.